docs/user/intro.rst

```diff
@@ -38,10 +38,11 @@ But there is more.. you can also give your module a dictionary in an interactive
 
 ::
 
-    >>> from mymodule import MyModule
+    >>> from argschema import ArgSchemaParser
+    >>> from mymodule import MySchema
     >>> d = {'a':5}
-    >>> mod = MyModule(input_data=d)
-    >>> mod.run()
+    >>> mod = ArgSchemaParser(input_data=d,schema_type=MySchema)
```

Comment:
It also appears to me that "input_data" is really a required argument.  The reason that argparse does not have this is because the input is assumed to be the command line.  I would actually be comfortable just reading "ArgSchemaParser(d, type=MySchema).